react-basics/State2/react-state-2_triple-count/src/components/Counter.jsx

```diff
@@ -1,14 +1,13 @@
 import { useState } from "react";
 import "./Counter.css";
 
-
 export default function Counter() {
   const [count, setCount] = useState(0);
 
   function handleIncrement() {
-    setCount(count + 1);
-    setCount(count + 1);
-    setCount(count + 1);
+    setCount((prev) => prev + 1);
```

Comment:
i thought that i have to use `prevCount` here, but nice that it works only with `prev`, didn't know that